sql/src/test/java/io/crate/planner/operators/SelectDistinctLogicalPlannerTest.java

```diff
@@ -59,11 +59,11 @@ public void testDistinctOnLiteralResultsInGroupByOnLiteral() {
         assertThat(plan, isPlan(e.functions(),
             "RootBoundary[[1, 2, 3]]\n" +
             "GroupBy[[1, 2, 3] | ]\n" +
-            "Collect[doc.users | [[1, 2, 3]] | All]\n"));
+            "Collect[doc.users | [[1, 2, 3]] | true]\n"));
     }
 
     @Test
-    public void testOrderByOnColumnNotPresentInDistinctOutputsIsNotAllowed() {
+    public void testOrderByOnColumnNotPresentInDistinctOutputsIsNottrueowed() {
```

Comment:
```suggestion
    public void testOrderByOnColumnNotPresentInDistinctOutputsIsNotTrue() {
```

Reply:
I `truowed` a typo ?

Reply:
Oops, that's the result of `%s/All/true/g`